subxt/src/backend/unstable/mod.rs

```diff
@@ -475,10 +524,8 @@ impl<T: Config + Send + Sync + 'static> Backend<T> for UnstableBackend<T> {
         let mut seen_blocks_sub = self.follow_handle.subscribe().events();
 
         // Then, submit the transaction.
-        let mut tx_progress = self
-            .methods
-            .transactionwatch_v1_submit_and_watch(extrinsic)
-            .await?;
+        let mut tx_progress =
+            retry(|| self.methods.transactionwatch_v1_submit_and_watch(extrinsic)).await?;
```

Comment:
As with the unstable backend, it's probably safer to not retry this for now, since retrying could lead to the user being returned `Invalid` while in fact the TX was actually submitted.